lib/disk/modules/VhdxDiskProbe.rb

```diff
@@ -0,0 +1,29 @@
+# encoding: US-ASCII
+$LOAD_PATH.push("#{File.dirname(__FILE__)}/../../Scvmm")
```

Comment:
@chessbyte that can be done but is there some guidance on when to update LOAD_PATH and when to use require_relative?  I see LOAD_PATH updates throughout MIQ.....